docs/conf.py

```diff
@@ -13,6 +13,7 @@
 
 import pydata_sphinx_theme
 from sphinx.application import Sphinx
+from sphinx.locale import _
```

Comment:
I'm not sure if this is the right import here

Reply:
> I'm not sure if this is the right import here

looks correct to me